apis/field_error.go

```diff
@@ -106,19 +105,31 @@ func (fe *FieldError) ViaFieldKey(field string, key string) *FieldError {
 func (fe *FieldError) Also(errs ...*FieldError) *FieldError {
 	newErr := &FieldError{}
 	// collect the current objects errors, if it has any
-	if fe != nil {
-		newErr.errors = fe.getNormalizedErrors()
+	if fe.isEmpty() == false {
```

Comment:
nit: why not `!fe.isEmpty()` ?

Reply:
(same below)

Reply:
I find the `== false` easier to read for these small cases

Reply:
@n3wscott fair enough :stuck_out_tongue_closed_eyes:

Reply:
I updated to BANG